packages/clay-css/src/scss/components/_print.scss

```diff
@@ -1,10 +1,7 @@
-// Source: https://github.com/h5bp/main.css/blob/master/src/_print.css
-
-// ==========================================================================
-// Print styles.
-// Inlined to avoid the additional HTTP request:
-// https://www.phpied.com/delay-loading-your-print-css/
-// ==========================================================================
+/* REUSE-Snippet-Begin
+* SPDX-License-Identifier: MIT
+* SPDX-FileCopyrightText: © 2018 HTML5 Boilerplate <https://github.com/h5bp/main.css>
+*/
```

Comment:
This looks whitespace damaged; would expect:

```
/* REUSE-Snippet-Begin
 * SPDX-License-Identifier: MIT
 * etc
```

Reply:
sent a SF pull at https://github.com/liferay/clay/pull/3485